api/projects/serializers.py

```diff
@@ -25,18 +29,20 @@ class Meta:
             "organisation",
             "hide_disabled_flags",
             "enable_dynamo_db",
-            "use_edge_identities",
```

Comment:
Hmm... this is going to be a breaking change on the frontend. Can we leave both fields in for now?